test/integration/agent/openflow_test.go

```diff
@@ -116,7 +116,7 @@ func TestConnectivityFlows(t *testing.T) {
 		antrearuntime.WindowsOS = runtime.GOOS
 	}
 
-	c = ofClient.NewClient(br, bridgeMgmtAddr, true, false, true, false, false, false, false)
+	c = ofClient.NewClient(br, bridgeMgmtAddr, true, false, true, false, false, false, false, false)
```

Comment:
could we add integration test for trafficcontrol?

Reply:
Added.